msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/msa/TestCase2637829.kt

```diff
@@ -45,10 +45,10 @@ import java.util.*
 @SupportedBrokers(brokers = [BrokerHost::class])
 @LocalBrokerHostDebugUiTest
 @RetryOnFailure
-class TestCase2637829 : AbstractMsaBrokerTest() {
+class TestCase2637829And2637846 : AbstractMsaBrokerTest() {
```

Comment:
Are these two tests similar? (in which case... we should probably just kill one of them from the test suite)

Reply:
Both test cases are covered in this test file

Reply:
I meant perhaps we can just get rid one of the cases. Not a blocker though.